OpenRA.Game/Graphics/VideoLoader.cs

```diff
@@ -15,15 +15,15 @@ namespace OpenRA.Video
 {
 	public interface IVideoLoader
 	{
-		bool TryParseVideo(Stream s, out IVideo video);
+		bool TryParseVideo(Stream s, bool useFramePadding, out IVideo video);
```

Comment:
Thought: Is it worth documenting the effects `useFramePadding`? I for one was surprised it's the max of both dimensions rounded up to nearest power of two, as opposed to each dimension individually rounded up to next power of 2.

e.g. Currently 100x10 gives a padding of 128x128, whereas individually would be 128x16.

Reply:
How/where would you recommend I document this though? It would probably have to be an XML comment on `IVideoLoader`, but it doesn't feel right there. *(Actually the padding is a special case for the `VideoPlayerWidget` sooo ugh...)*